R/runs.R

```diff
@@ -14,14 +14,37 @@ metric_value_to_rest <- function(value) {
   }
 }
 
-#' Log Metric
+#' @importFrom tibble tibble
+#' @importFrom rlang names2
+get_key_value_df <- function(...) {
+  values <- list(...) %>% unlist()
+  keys <- names2(values)
+  args <- as.list(sys.call(-1))
+  fname <- args[[1]]
+  if(fname == "log_metrics") {
+    values <- values %>% map_chr(metric_value_to_rest)
+  }
```

Comment:
any objections to this @mrkaye97 ? it works

Reply:
I think we want `fname == "log_params"`